Formula/gops.rb

```diff
@@ -0,0 +1,18 @@
+class Gops < Formula
+  desc "Tool to list and diagnose Go processes currently running on your system"
+  homepage "https://github.com/google/gops"
+  url "https://github.com/google/gops/archive/refs/tags/v0.3.22.tar.gz"
+  sha256 "a8092305465b28b0db83eb7087edca958de6522bc3ebb14656ecb8aef521e07b"
+  license "BSD-3-Clause"
+  head "https://github.com/google/gops.git", branch: "master"
+
+  depends_on "go" => :build
+
+  def install
+    system "go", "build", *std_go_args(ldflags: "-s -w")
+  end
+
+  test do
+    assert_match "gops: Missing PID or address.", shell_output("#{bin}/gops stats", 1)
```

Comment:
Could we so a test similar to the one in `pidof`? Except the test program compiled will be in Go instead of C.

Reply:
let me give a shot.